src/frontend/screens/Settings/sections/SystemInfo/index.tsx

```diff
@@ -0,0 +1,140 @@
+import Typography from '@mui/material/Typography'
+import Button from '@mui/material/Button'
+import Box from '@mui/material/Box'
+import { StyledEngineProvider } from '@mui/material/styles'
+
+import { useAwaited } from 'frontend/hooks/useAwaited'
+
+import { ReactComponent as SteamDeckLogo } from 'frontend/assets/steam-deck-logo.svg'
+import ContentCopyIcon from '@mui/icons-material/ContentCopy'
+
+import CPUCard from './cpu'
+import MemoryProgress from './memory'
+import GPUCard from './gpu'
+import OSInfo from './os'
+import SoftwareInfo from './software'
+
+import './index.scss'
+
+import type { SystemInformation } from 'backend/utils/systeminfo'
+
+interface SystemSpecificationsProps {
+  systemInformation: SystemInformation
+}
+
+function SystemSpecifications({
+  systemInformation
+}: SystemSpecificationsProps) {
+  return (
+    <Grid container spacing={1}>
+      <Grid item xs={6}>
+        <CPUCard cpu={systemInformation.CPU} />
+      </Grid>
+      <Grid item xs={6}>
+        <MemoryProgress memory={systemInformation.memory} />
+      </Grid>
+      {...systemInformation.GPUs.map((gpu, index) => (
+        <Grid key={index} item xs={6}>
+          <GPUCard
+            gpu={gpu}
+            gpuNumber={index}
+            showNumber={systemInformation.GPUs.length !== 1}
+          />
+        </Grid>
+      ))}
+    </Grid>
+  )
+}
+
+function SteamDeckSystemSpecifications({
+  systemInformation
+}: SystemSpecificationsProps) {
+  const { t } = useTranslation()
+
+  return (
+    <>
+      <Paper sx={{ width: '50%' }} square>
+        <Typography variant="h6">
+          {t('settings.systemInformation.systemModel', 'System Model:')}
+        </Typography>
+        <Grid container>
+          <Grid item xs={2}>
+            <SteamDeckLogo className="logo fillWithThemeColor" />
+          </Grid>
+          <Grid item xs={10}>
+            {t('settings.systemInformation.steamDeck', 'Steam Deck')}
+          </Grid>
+        </Grid>
+      </Paper>
+      <details>
+        <summary className="showSystemSpecifications">
+          {t(
+            'settings.systemInformation.showDetailed',
+            'Show detailed system specifications'
+          )}
+        </summary>
+        <SystemSpecifications systemInformation={systemInformation} />
+      </details>
+    </>
+  )
+}
+
+export default function SystemInfo() {
+  const { t } = useTranslation()
+
+  const systemInformation = useAwaited(async () =>
+    window.api.systemInfo.get(false)
+  )
+  if (!systemInformation) return <CircularProgress />
+
+  return (
+    <StyledEngineProvider injectFirst>
+      <Box sx={{ width: '770px', textAlign: 'start' }}>
+        <>
+          <h3>
+            {t('settings.navbar.systemInformation', 'System Information')}
+          </h3>
+          <h5>
+            {t(
+              'settings.systemInformation.systemSpecifications',
+              'System Specifications:'
+            )}
+          </h5>
```

Comment:
Maybe nitpicky here, but we have the h3 and h5 sounding redundant
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/988adc42-6ba2-4050-948a-a8af0061ee84)

Maybe we can just leave the h3? unless there's a reason to put that h5 there (maybe the idea is to put more content?)

Reply:
I originally had clarity in mind with this (your CPU, memory, and GPUs are your *specifications*, while everything on the page in general is *information*), but I'd be fine with removing it as well